### ISSUE TITLE
Change VM details Disks card disk sort order

### ISSUE THREAD

**Issue by sjd78** [oVirt/ovirt-web-ui#1146]
> > > 1. Displayed disks order. Should we always jump bootable disks to first in line and avoid disks naming order? I think it's confusing (so we can open a separate issue on that and handle that later on).
>
> A new issue is the way to go for the sort on the disks card.

_Originally posted by @sjd78 in https://github.com/oVirt/ovirt-web-ui/pull/1132#issuecomment-561291986_

On the VM Details, Disks card, the bootable disk is sorted before any other disk.  If the bootable flag is changed by the user (once PR #1132 is merged), the disks will be resorted.  To avoid confusion with the sorting change, simply sort the disks by disk name.  The "bootable" label should be enough to easily identify which disk is the VMs bootable disk.
This is the default behaviour on webadmin as well.

For example, the disks list are curenttly displayed on web_ui like this:
![VM Portal-disks-ordering](https://user-images.githubusercontent.com/18169498/75987230-458d2600-5ef8-11ea-8289-c8c44dfc8fef.png)

While on webadmin, exactly the same disks will be sorted by name as a default:
![webadmin-disks-ordering](https://user-images.githubusercontent.com/18169498/75987322-6a819900-5ef8-11ea-98db-5712b3ca67b2.png)
